### ISSUE TITLE
arcade.draw_text: letters are cutting

### ISSUE THREAD

**Issue by alderven** [pythonarcade/arcade#937]
Run code:
```
import pyglet
import arcade

arcade.open_window(600, 400, 'Hello')
pyglet.font.add_file(r'C:\propisi.ttf')
arcade.draw_text(text='УРОДОВИЧ!', start_x=300, start_y=200, font_size=100, color=arcade.color.RED, font_name='propisi', anchor_x='center', anchor_y='center')
arcade.run()
```

Arcade 2.6.0 — right side of the letters is cutting:

![изображение](https://user-images.githubusercontent.com/21060932/131736806-5891fad9-17e4-4e69-9fb1-32ce8cf0d7a7.png)

Arcade 2.5.7 — no errors:

![изображение](https://user-images.githubusercontent.com/21060932/131736540-4e616201-e8c6-4a8e-aef4-90cf452dc1d5.png)

Python: 3.8
Arcade: 2.6.0
OS: Win 10

[propisi.zip](https://github.com/pythonarcade/arcade/files/7096286/propisi.zip)

**Comment by einarf** [comment]
That looks like a blending issue. Are we disabling blending at some points somewhere?

What happens if you only include the first two characters?

**Comment by einarf** [comment]
It's an issue with GDI Plus in pyglet. If you can provide the ttf file it would be a lot easier to debug.

You can also use `arcade.create_text_sprite` or `arcade.create_text_image`. This is still using the old pillow text rendering.

**Comment by alderven** [comment]
> If you can provide the ttf file it would be a lot easier to debug.

[propisi.zip](https://github.com/pythonarcade/arcade/files/7096286/propisi.zip)

**Comment by einarf** [comment]
Some tweaks in pyglet by @caffeinepills seems to fix this
![bilde](https://user-images.githubusercontent.com/5418180/133022985-7e7da29d-c101-4093-9b10-b7b47efc88fa.png)

**Comment by einarf** [comment]
Upgrade pyglet to 2.0dev8

**Comment by einarf** [comment]
2,6,2 will solve this. As mentioed above you can also manually upgrade to pylget==2.0dev8

**Comment by alderven** [comment]
Upgraded Arcade to `2.6.2` and pyglet to `2.0dev8` (on Python `3.8`) but don't see any difference:

![изображение](https://user-images.githubusercontent.com/21060932/134117202-b67d1f10-0e53-452e-93e5-b37db1251429.png)

**Comment by einarf** [comment]
It seems the fix was not merged into pyglet master in time. I've poked and we'll just have to wait a little bit longer.

**Comment by einarf** [comment]
Works with pyglet 2.0dev9. We'll do a 2.6.3 release shortly using this.

**Comment by einarf** [comment]
Fixed in 2.6.3. Be free to reopen
